api/src/main/resources/messages.properties

```diff
@@ -56,6 +56,12 @@ coreapps.task.mergeVisits.mergeSelectedVisits=Merge Selected Visits
 coreapps.task.mergeVisits.success=Visits merged successfully
 coreapps.task.mergeVisits.error=Failed to merge visits
 
+coreapps.task.deletePatient.label=Delete Patient
+coreapps.task.deletePatient.notAllowed=User does not have permission to delete patient
+coreapps.task.deletePatient.deletePatientSuccessful=Patient has been successfully deleted
```

Comment:
What issue?
On 11-May-2016 4:09 AM, "Wyclif Luyima" <notifications@github.com> wrote:

> In api/src/main/resources/messages.properties
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20#discussion_r62766414>
> :
>
> > @@ -56,6 +56,12 @@ coreapps.task.mergeVisits.mergeSelectedVisits=Merge Selected Visits
> >  coreapps.task.mergeVisits.success=Visits merged successfully
> >  coreapps.task.mergeVisits.error=Failed to merge visits
> >
> > +coreapps.task.deletePatient.label=Delete Patient
> > +coreapps.task.deletePatient.notAllowed=User does not have permission to delete patient
> > +coreapps.task.deletePatient.deletePatientSuccessful=Patient has been successfully deleted
>
> You can actually go ahead to fix this message too as you fix the void
> reason issue too
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20/files/1c5f49afd815098d3cc2787220675efad16f3bd2#r62766414>
>